optimum/intel/openvino/__init__.py

```diff
@@ -22,3 +22,5 @@
 )
 from .modeling_seq2seq import OVModelForSeq2SeqLM
 from .utils import OV_DECODER_NAME, OV_DECODER_WITH_PAST_NAME, OV_ENCODER_NAME, OV_XML_FILE_NAME
+
+from .nncf_config import DEFAULT_QUANTIZATION_CONFIG
```

Comment:
@echarlaix @AlexKoff88 Can we sync on the import location? With this PR DEFAULT_QUANTIZATION_CONFIG is imported `from optimum.intel.openvino` but OVQuantizer is imported `from optimum.intel.openvino.quantization`. They should have the same root. Since optimum.intel.openvino is already three levels, I'm happy with everything under that, but open to other ideas.

Reply:
Sure